k8s/aat/ccd/data-store-api/data-store-api.yaml

```diff
@@ -20,6 +20,7 @@ spec:
       applicationInsightsInstrumentKey: "f61ab34c-07da-42fc-ac7e-3f8cdadfd5a1"
       image: hmcts.azurecr.io/hmcts/ccd-data-store-api:latest-1ab5c70c
       ingressHost: ccd-data-store-api.service.core-compute-aat.internal
+      secrets:
```

Comment:
did you change the right file? this PR talks about blob storage being disabled

Reply:
🤦‍♂️